lib/versioning/node/index.spec.ts

```diff
@@ -46,14 +59,68 @@ describe('versioning/node/index', () => {
       ${'10.1.0'}   | ${t1} | ${true}
       ${'10.0.0a'}  | ${t1} | ${false}
       ${'9.0.0'}    | ${t1} | ${false}
-    `('isStable("$version") === $expected', ({ version, time, expected }) => {
+    `('("$version") === $expected', ({ version, time, expected }) => {
```

Comment:
```suggestion
    `('isStable("$version") === $expected', ({ version, time, expected }) => {
```

Reply:
Hm, I changed that based on @viceice suggestion in https://github.com/renovatebot/renovate/pull/13466#discussion_r781511853 😀

But no problem, I'll get rid of the extra describe sections anyway, then we have the function name here again.

Reply:
https://github.com/renovatebot/renovate/pull/13466/commits/e41d17c9f028a1f370b15d81e4ac94de2746ae2b